SpatialGDK/Source/SpatialGDKEditorCommandlet/Private/Commandlets/GenerateSchemaAndSnapshotsCommandlet.cpp

```diff
@@ -49,25 +49,34 @@ int32 UGenerateSchemaAndSnapshotsCommandlet::Main(const FString& Args)
 		FString RemainingMapPaths = MapNameParam;
 		while (RemainingMapPaths.Split(TEXT(";"), &ThisMapName, &RemainingMapPaths))
 		{
-			GenerateSchemaAndSnapshotForPath(SpatialGDKEditor, ThisMapName);
+			if (!GenerateSchemaAndSnapshotForPath(SpatialGDKEditor, ThisMapName))
+			{
+				return 1;	// Error
```

Comment:
Would we want to make 0 and 1 into constant variables with names like SUCCESS and ERROR?

Reply:
I feel like that is making unnecessary work unless you intend to support more than just 0 and 1, don't you think? It's a pretty known standard that return code 0 is success and 1 is error, methinks.

Reply:
Sure, was just thinking it would be for readability.